src/translations/en.json

```diff
@@ -511,6 +512,9 @@
   "NoAccessPage.postListings.schemaTitle": "No publishing rights",
   "NoAccessPage.postListings.heading": "You can't publish listings",
   "NoAccessPage.postListings.content": "You need to receive publishing rights from the {marketplaceName} team.",
+  "NoAccessPage.initiateTransactions.schemaTitle": "No transaction rights",
+  "NoAccessPage.initiateTransactions.heading": "You don’t have transaction rights",
```

Comment:
The apostrophe should be `'` (instead of `’`), so that it is consistent with other places we use this symbol.

Reply:
Fixed!